spec/purger_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe Purger do
+  let(:url) { "https://www.gov.uk/government/news/govuk-implements-new-cache-clearing" }
+
+  subject { described_class.new(Logger.new("/dev/null")) }
+
+  context "if the purge request succeeds" do
+    before do
+      stub_request(:purge, /cache/)
+    end
+
+    it "purges the varnish cache for the base path in the payload" do
```

Comment:
Should this now be generic, i.e. not "varnish"?

Reply:
Yep, I'll fix that.